server/backend/database/memory/indexes.go

```diff
@@ -19,13 +19,14 @@ package memory
 import "github.com/hashicorp/go-memdb"
 
 var (
-	tblProjects   = "projects"
-	tblUsers      = "users"
-	tblClients    = "clients"
-	tblDocuments  = "documents"
-	tblChanges    = "changes"
-	tblSnapshots  = "snapshots"
-	tblSyncedSeqs = "syncedseqs"
+	tblProjects      = "projects"
+	tblUsers         = "users"
+	tblClients       = "clients"
+	tblDocuments     = "documents"
+	tblChanges       = "changes"
+	tblSnapshots     = "snapshots"
+	tblSyncedSeqs    = "syncedseqs"
+	tblVersionVector = "versionvector"
```

Comment:
It would be good to fit this into convention.
```suggestion
	tblVersionVectors = "versionvectors"
```